machineSetup/setupPerfMachine.ps1

```diff
@@ -0,0 +1,144 @@
+﻿#Set power scheme to high performance
```

Comment:
($ErrorActionPreference may or may not be a good idea, depending on whether things we call write to stderr or not; but without it, we should pass a lot more error checking calls, like `-ErrorAction 'Stop'` to calls like mkdir, etc.)